bullet_train/config/locales/en/addresses.en.yml

```diff
@@ -0,0 +1,128 @@
+en:
+  addresses:
+    label: "Address"
+    contact: "Contact"
+    fields:
+      address_one:
+        _: &address_one Address
+        label: *address_one
+        heading: *address_one
+      address_two:
+        _: &address_two Address cont.
+        label: *address_two
+        heading: *address_two
+      city:
+        _: &city City
+        label: *city
+        heading: *city
+      region_id: &region
+        _: &region_id State / Province / Region
+        label: *region_id
+        heading: *region_id
+        options:
```

Comment:
We should populate this field from `Address::Region` vs. defining them like this.

Reply:
I'm adding a new "admin_divisions" entry on each country, which is a key to use in the locale file (there won't be too many) e.g. (regions, provinces_territories, states, prefectures). There's a [table on wikipedia of how each country names its administrative divisions](https://en.wikipedia.org/wiki/List_of_administrative_divisions_by_country) from which we can populate those keys to be localized for the label.

Reply:
Removed the options